core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/PlayerOnErrorForwarder.java

```diff
@@ -0,0 +1,57 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ExoPlayer;
+import com.google.android.exoplayer2.PlaybackParameters;
+import com.google.android.exoplayer2.Timeline;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
+import com.novoda.noplayer.Player;
+import com.novoda.noplayer.listeners.ErrorListeners;
+
+class PlayerOnErrorForwarder implements ExoPlayer.EventListener {
+
+    private final Player player;
+    private final ErrorListeners errorListeners;
+
+    PlayerOnErrorForwarder(Player player, ErrorListeners errorListeners) {
+        this.player = player;
+        this.errorListeners = errorListeners;
+    }
+
+    @Override
+    public void onPlayerError(ExoPlaybackException error) {
+        Player.PlayerError playerError = ExoPlayerErrorFactory.errorFor(error);
+        errorListeners.onError(player, playerError);
+    }
+
+    @Override
+    public void onTimelineChanged(Timeline timeline, Object manifest) {
+        //TODO should we send ?
```

Comment:
AND AGAIN! 😄 🎉  We need to figure out the mapping between `exoplayer` and `no-player`.